doc/Makefile

```diff
@@ -126,6 +128,8 @@ pdf: $(OBJECTS) $(ANCHORCHECK)
                 sphinx-build $(SPHINXEXTRA) -b latex -c utils/sphinx-config -d $(BUILDDIR)/doctrees $(RSTDIR) latex ;\
                 echo "############################################" ;\
                 rst_anchor_check src/*.rst ;\
+		env LC_ALL=C grep -n '[^ -~]' $(RSTDIR)/*.rst ;\
```

Comment:
indentation looks a bit weird

Reply:
blanks vs. tabs. need to update my emacs settings to show tabs as i have it in vi. going all-tabs now, also for other targets.